### PR TITLE
OLMIS-7649: Prepare administartor page for data export

### PR DESCRIPTION
**Pull request by sniedzielski**
TICKET: https://openlmis.atlassian.net/browse/OLMIS-7649

NOTE: 
This PR contains solution without sending request to backend and without toast after downloading mocked zip file. For that moment this solution generated mocked ZIP file until backend API endpoint is ready. This will be done in scope of ticket https://openlmis.atlassian.net/browse/OLMIS-7705

I marked TODO comments to indicate which part of code could be changed/removed when backend is ready.

Presentation of implemented solution:
Peek 2023-01-20 16-09

### CODE REVIEW

File: src/admin-data-export/admin-data-export.routes.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        $stateProvider.state('openlmis.administration.dataExport', {
+            url: '/dataExport',
+            label: 'admin.dataExport.label',
+            isOffline: false,
+            priority: 11,
+            showInNavigation: true,
+            views: {
+                '@': {
+                    templateUrl: 'admin-data-export/admin-data-export.html'
+                }
```
Comment (NikolaLaskowska) [inline src/admin-data-export/admin-data-export.routes.js:35]: could you add a TODO line for `accessRights:`?

Comment (sniedzielski) [reply src/admin-data-export/admin-data-export.routes.js:35]: yes, sure. I can add.

File: src/admin-data-export/consts.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+ * http://www.gnu.org/licenses.  For additional information contact info@OpenLMIS.org. 
+ */
+
+export const TYPE_OF_EXPORTS = [
+    {
+        value: "Program",
+        name: "Program"
+    },
+    {
+        value: "Facility",
```
Comment (NikolaLaskowska) [inline src/admin-data-export/consts.jsx:22]: for now we won't support the facility files export

File: src/admin-data-export/consts.jsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+            "id": "ProgramOrderables.csv",
+            "name": "ProgramOrderables.csv"
+        },
+        {
+            "type": "Program", 
+            "id": "TradeItems.csv",
+            "name": "TradeItems.csv"
+        },
+        {
+            "type": "Facility",
```
Comment (NikolaLaskowska) [inline src/admin-data-export/consts.jsx:48]: for now we won't support the facility files export

File: src/admin-data-export/consts.jsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+            "id": "TradeItems.csv",
+            "name": "TradeItems.csv"
+        },
+        {
+            "type": "Facility", 
+            "id": "Facilities.csv",
+            "name": "Facilities.csv"
+        },
+        {
+            "type": "Facility",
```
Comment (NikolaLaskowska) [inline src/admin-data-export/consts.jsx:53]: for now we won't support the facility files export

File: src/admin-data-export/consts.jsx [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+            "name": "Facilities.csv"
+        },
+        {
+            "type": "Facility", 
+            "id": "Test.csv",
+            "name": "Test.csv"
+        }
+];
+
+export const SELECT_OPTION_LABEL = "Select an option";
```
Comment (NikolaLaskowska) [inline src/admin-data-export/consts.jsx:59]: I think it is not necessary

**Comment by NikolaLaskowska** [comment]
Please add a changelog entry

**Review by NikolaLaskowska — APPROVED** [review]

**Pull request merged**